src/web/RootView.tsx

```diff
@@ -462,9 +463,9 @@ export class RootView extends React.Component<RootViewProps, RootViewState> {
     private _startHidePopupTimer() {
         if (this.props.autoDismiss) {
             // Should we immediately hide it, or did the caller request a delay?
-            if (this.props.autoDismissDelay > 0) {
+            if (this.props.autoDismissDelay && this.props.autoDismissDelay > 0) {
```

Comment:
Zero values work OK with this logic, but an explicit undefined type would be better.